bluemira/equilibria/find_legs.py

```diff
@@ -234,34 +248,52 @@ def get_legs(
         return leg_dict
 
 
-def get_legs_length(
+def get_legs_length_and_angle(
     eq: Equilibrium,
     leg_dict: dict,
     plasma_facing_boundary: Grid | Coordinates | None = None,
 ):
     """Calculates the length of all the divertor legs in a dictionary."""
     length_dict = {}
-    for n, leg_list in leg_dict.items():
-        if not isinstance(leg_list[0], Coordinates):
+    angle_dict = {}
+    for name, leg_list in leg_dict.items():
+        if not leg_list:
             lengths = [0.0]
+            angles = [np.pi]
```

Comment:
This is not hit in the tests because you've set the leg length to [None] and not None. Is that intentional?

Reply:
Good spot, this is out of date, have modified